tests/ios/InAppBrowserTests/InAppBrowserTestsTests/CDVArgumentsParserTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CordovaArgumentsParserTests.swift
+//  InAppBrowserTestsTests
+//
+//  Created by Goncalo Frade on 10/08/2020.
+//  Copyright © 2020 Goncalo Frade. All rights reserved.
```

Comment:
I believe this is something that you have by default but please review it for our context

Reply:
Ah yes. Must have escaped I usually delete those they are not needed. Also updated to version OS1 in all the packages and plugin.xml.